dsl_parser/elements/policies.py

```diff
@@ -230,7 +241,7 @@ class Group(DictElement):
 
     schema = {
         'members': GroupMembers,
```

Comment:
No constant for `members`?

Reply:
Sadly no, I was not sure if to go on to create those constants or this not productive work